public/content/it/index.yml

```diff
@@ -0,0 +1,4 @@
+title: Il tour di D
```

Comment:
It sounds bad to me "di D".  "di" and "D" are read in the same way in italian. Maybe "Il tour del linguaggio D?"